### DISCUSSION TITLE
Community of Peers | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#72] [Comments]
# Community of Peers | 夜天之书

Community of Peers 可以翻译成『同侪社群』，意即社群由地位相同的成员组成，这是 Apache 软件基金会（ASF）的开源理念之一。 ASF 强调个体组成社群，每个成员代表他们自己而不是其所属的组织。不管成员是否受雇参与贡献，社群视角下他们都是一视同仁的志愿者。尽管只有 Committer 才有合并代码的权限，只有项目管理委员会（PMC）成员才能对项目的关键决议投有效票，但是除此以

https://www.tisonkun.org/2023/03/27/community-of-peers/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

> ...下意识觉得这是在给自己派活儿

这简直忒真相了, 思想钢印了都;
其实, 最大的问题是指派这个行为的流程出了问题?

社区和公司中, 最大的区别可能就是共识形成的流程正义完全不同?

虽然, 社区中的事儿, 应该都是 我想/我要, 而不是"我必须","只能我"...

可是, 我想 这个意愿得先表达出来,
否则, 指派这个行为包含的其它善意, 就会被指派这个组织流程仪式给反转了...

也就是说无论 『和而不同』与『周而不比』 这其中的 和以及周 都是要全体事先进行确认的;
否则, 在缺席情况下被代表定义为  和以及周 当然会引发额外的情绪...
